### PR TITLE
Nightlies slack integration

### LINKED ISSUE #2403: Send notification of results of nightly ci tests
We recently added some nightly jobs to our CI in #2250 ! 

The next step is set up a notification with the success or failure of the nightlies and a link to view the github pipeline.

I'm open to discussing what the medium for the notification should be but this looks interesting: https://github.com/marketplace/actions/slack-notify

### PR DESCRIPTION
**Pull request by bchen1116**
fix #2403 

Check that slack for results

[This](https://action-slack.netlify.app/) is the package I used

### CODE REVIEW

File: .github/workflows/windows_nightlies.yml [hunk 1]
```diff
@@ -69,4 +69,16 @@ jobs:
         run: |
           . $env:USERPROFILE\Miniconda3\shell\condabin\conda-hook.ps1
           conda activate curr_py
-          make ${{matrix.command}}
+          make ${{matrix.command}}]
```
Comment (freddyaboulton) [inline .github/workflows/windows_nightlies.yml:72]: What's up with the right square bracket?

**Review by freddyaboulton — APPROVED** [review]
Looks good to me @bchen1116 !! :elmo-fire: 😂

**Review by jeremyliweishih — APPROVED** [review]
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

File: .github/workflows/windows_nightlies.yml (hunk 1 above)
Comment (bchen1116) [reply .github/workflows/windows_nightlies.yml]: Ah that's an accident. Good catch!

**Pull request merged**